src/main/java/com/zufar/onlinestore/payment/dto/PaymentDetailsWithTokenDto.java

```diff
@@ -4,5 +4,4 @@
 
 @Builder
 public record PaymentDetailsWithTokenDto(String paymentToken,
-                                         PaymentDetailsDto paymentDetailsDto) {
-}
+                                         PaymentDetailsDto paymentDetailsDto) {}
```

Comment:
please, move the curly bracket to the new line
```
  ...
PaymentDetailsDto paymentDetailsDto) {
  }
```

Reply:
done